treeherder/etl/classification_loader.py

```diff
@@ -88,3 +98,18 @@ def get_push(self, task_route):
             raise
 
         return push
+
+    def autoclassify_failures(self, failures):
+        autoclassified_intermittent = FailureClassification.objects.get(
+            name="autoclassified intermittent"
```

Comment:
This will raise if the failure classification is not available: you should handle that case with a try/except + logger.error here